From 72d145f0bc9f46a1302a288e6cf989947eb5f068 Mon Sep 17 00:00:00 2001
From: Anu Engineer <aengineer@apache.org>
Date: Thu, 3 Mar 2016 21:27:57 -0800
Subject: [PATCH 1543/2848] HDFS-9856. Suppress Jenkins warning for sample
 JSON file. Contributed by Xiaobing Zhou

(cherry picked from commit c5708a1a07f187f4d192ea44c33af39150e6163a)

Change-Id: Ic10c820c6d4d16acd0e19746ed496bba5e0dd585
---
 hadoop-hdfs-project/hadoop-hdfs/pom.xml |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
index 96bb268..9dee97e 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
@@ -550,6 +550,8 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
             <exclude>src/main/webapps/static/json-bignum.js</exclude>
             <exclude>src/main/webapps/static/dataTables.bootstrap.css</exclude>
             <exclude>src/main/webapps/static/dataTables.bootstrap.js</exclude>
+            <exclude>src/test/resources/diskBalancer/data-cluster-3node-3disk.json</exclude>
+            <exclude>HDFS-1312_CHANGES.txt</exclude>
           </excludes>
         </configuration>
       </plugin>
-- 
1.7.9.5

